### ISSUE TITLE
Saving to .svg on Windows 10: JSONDecodeError

### ISSUE THREAD

**Issue by gregorhd** [altair-viz/altair_saver#97]
Hi there, even after reading the instructions multiple times, I'm still not able to save a geoshapes-based chart to .svg with `altair_saver ` on Windows 10 (executing in VSCode-Python). I'm getting this error [reported elsewhere](https://github.com/altair-viz/altair_saver/issues/95) when calling both `chart.save("test.svg", webdriver='firefox')` as well as `chart.save("test.svg", webdriver='chrome')`:

`JSONDecodeError: Expecting value: line 2 column 1 (char 2)`

My relevant package versions are

```
altair 4.1.0
altair_saver 0.5.0
vega 3.5.0
vega-cli 5.17.0             
vega-lite-cli 4.17.0               
selenium 3.141.0
python-chromedriver-binary 92.0.4515.43.0
```
My Chrome browser is version 92.0.4515.131.

I added the path to a chromedriver.exe to the `Path` environment variable (in the system variables). Calling `print(os.environ)` from within my Python environment also shows this path clearly being listed.

I know this package hasn't been tested for Windows, but Is there anything else I could try to get this to work?

Thanks a million!

**Comment by sparisi** [comment]
I am having the exact same problem. Any news on that? @gregorhd did you find a solution? Thanks!

**Comment by gregorhd** [comment]
Hi Simone, unfortunately not. I just gave up and ended up saving a low-res png directly from the browser… :-/

**Comment by sparisi** [comment]
Got it, thanks anyway.
Btw, I partially fixed the low-quality problem by manually setting height and width of my chart to some large values, but I have to manually rescale everything else as well (labels, fonts, ticks, ...). And it doesn't work in some cases (for example with `resolve_scale(x='independent')`. 
It is something, but saving directly in high-res would be much better .

**Comment by gregorhd** [comment]
Ah good to know! I’ll try that as a workaround next time. Cheers!

**Comment by Tigxy** [comment]
Hi there!
Had the same problem, but the answer to a [related issue](https://github.com/altair-viz/altair/issues/2062#issuecomment-1143993037) has helped me fix it: 
> try `npm install -g vega vega-cli vega-lite canvas`

**Comment by joelostblom** [comment]
Since Altair 5.2, the functionality of Altair Saver is now available in Altair via the [vl-convert](https://github.com/vega/vl-convert) package. Most of the functionality has been available since 5.0, and [the main addition in 5.2 was PDF export](https://github.com/altair-viz/altair/pull/3244). See [the docs on how to save charts](https://altair-viz.github.io/user_guide/saving_charts.html#png-svg-and-pdf-format) for more details

We are going to archive this repo, so I'm closing all the open issues and PRs before doing so. Try out the new options for saving charts mentioned above and if you run into issues, please open an issue directly in the altair or vl-convert repo.
